modules/home/assets/js/components/HomeScreen.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
```

Comment:
I believe we use` lower-case.js` filenames everywhere in the Elementor repositories. Can you please verify that for me? 
I think it would be better to stick to `home-screen.js`.

Reply:
done, in announcement it was in Capital, that got me confused